pkg/util/kubernetes/kubelet/kubelet_test.go

```diff
@@ -597,15 +597,12 @@ func (suite *KubeletTestSuite) TestKubeletInitTokenHttps() {
 	<-k.Requests // Throwing away first GET
 
 	assert.Equal(suite.T(), fmt.Sprintf("https://127.0.0.1:%d", kubeletPort), ku.kubeletClient.kubeletURL)
-	assert.Equal(suite.T(), "bearer fakeBearerToken", ku.kubeletClient.headers.Get("Authorization"))
-	assert.True(suite.T(), ku.kubeletClient.client.Transport.(*http.Transport).TLSClientConfig.InsecureSkipVerify)
```

Comment:
`transport.NewBearerAuthWithRefreshRoundTripper` returns the `http.RoundTripper` interface, and the implementation is not public - it's impossible to cast here.
This assertion should be covered anyway by these lines below
```golang
	assert.Equal(suite.T(), "ok", string(b))
	assert.Equal(suite.T(), 200, code)
```